evergreen/evergreen.yml

```diff
@@ -217,6 +217,16 @@ functions:
           ${PREPARE_SHELL}
           AUTH=${AUTH} SSL=${SSL} MONGODB_URI="${MONGODB_URI}" TOPOLOGY=${TOPOLOGY} OS=${OS} evergreen/run-tests.sh
 
+  run-atlas-connectivity-tests:
+    - command: shell.exec
+      type: test
+      params:
+        silent: true
+        working_dir: mongo-csharp-driver
+        script: |
+          # DO NOT ECHO WITH XTRACE (which PREPARE_SHELL does)
```

Comment:
I think the whole xtrace thing is a red herring here because we pass the URLs in environment variables so they never would have been logged anyway (other than by Evergreen itself but see silent: true).

But no harm in emphasizing that we don't want these commands logged.